<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-HDPU2L2" failures="1" tests="1" name="Test(failed)" time="0.07" errors="0" timestamp="2024-05-12T04:21:05 IST">
  <testcase classname="sdet.SecondTest" name="login" time="0.02">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;1&amp;gt; but was:&amp;lt;0&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<1> but was:<0>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:201)
at junit.framework.Assert.assertEquals(Assert.java:207)
at sdet.SecondTest.login(SecondTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="sdet.SecondTest" name="setUp" time="0.0">
    <ignored/>
  </testcase> <!-- setUp -->
  <testcase classname="sdet.SecondTest" name="close" time="0.0">
    <ignored/>
  </testcase> <!-- close -->
</testsuite> <!-- Test(failed) -->
